frontend/src/v5/ui/routes/dashboard/projects/federations/editFederationModal/editFederation/editFederationContainersList/editFederationContainersListItem/editFederationContainersListItem.styles.ts

```diff
@@ -28,3 +29,21 @@ export const DashboardListItemRow = styled(DashboardListItemRowBase)`
 		margin-right: 20px;
 	}
 `;
+
+export const Autocomplete = styled(AutocompleteBase)`
```

Comment:
![image](https://github.com/3drepo/3drepo.io/assets/62481250/8f2fc68b-d9fd-4f90-99a5-d0093483676a)
There's a shadow floating when the list is empty